data/json/monstergroups/wilderness.json

```diff
@@ -1814,7 +1814,7 @@
       { "monster": "mon_nakedmolerat_giant", "weight": 1, "cost_multiplier": 3, "conditions": [ "DUSK", "NIGHT" ] },
       { "monster": "mon_mutant_experimental", "weight": 3, "cost_multiplier": 0, "conditions": [ "DUSK", "NIGHT" ] },
       { "monster": "mon_mantis_small", "weight": 8, "cost_multiplier": 10 },
-      { "monster": "mon_lady_bug_giant", "weight": 5, "cost_multiplier": 10 },
```

Comment:
Add this spawn back with the start time

Reply:
I thought ladybugs evolve into giant ones with the halflife of 14 days?